docs/basics/column.md

```diff
@@ -148,8 +148,8 @@ def my_func(df):
     print(my_func(df))
     ```
 
-Note that, this time, we couldn't use `@nw.narwhalify`, as the final step in
-our function wasn't `nw.to_native`, so we had to explicitly use `nw.from_native`
-as the first step. In general, we recommend using the decorator where possible,
-as it looks a lot cleaner, and only using `nw.from_native` / `nw.to_native` explicitly
-when you need them.
+Note that, even though the output of our function is not a dataframe nor a series, we can
+still use `narwhalify` with `strict=False` flag (which is actually the default behavior).
```

Comment:
maybe we should just show the default? this is probably one of the first pages newcomers will see, maybe it's better to keep things simple?